src/GitHub.TeamFoundation.14/Sync/GitHubPublishSection.cs

```diff
@@ -113,7 +113,7 @@ public void ShowPublish()
 
             controller.TransitionSignal.Subscribe(data =>
             {
-                var c = data.View;
+                var c = (IDialogView)data.View;
```

Comment:
`IHasBusy` only has a `bool HasBusy` property, not an observable. I could use `.WhenAny` here but this is already addressed in #933 so might as well leave it for now?